pynamodb/attributes.py

```diff
@@ -493,8 +493,11 @@ def deserialize(self, values):
             attr_value = _get_value_for_deserialize(v)
             key = self._dynamo_to_python_attr(k)
             attr_class = self._get_deserialize_class(key, v)
+            deserialized_value = None
```

Comment:
since you know the attr_value is gonna be null in line 493, it is easier to read by doing

```
if not attr_value:
    continue
```